roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1425,7 +1425,7 @@ def get_defaults(self, roles, deployment_type, deployment_subtype):
                                       console_use_ssl=True,
                                       console_path='/console',
                                       console_port='8443', etcd_use_ssl=True,
-                                      etcd_hosts='', etcd_port='4001',
+                                      etcd_hosts='', etcd_port='2379',
```

Comment:
This is good for now.

We also need to get this to align with the other variables that are used outside of openshift_facts